opsdroid/connector/matrix/connector.py

```diff
@@ -214,13 +252,8 @@ async def _parse_sync_response(self, response):
             if roomInfo.timeline:
                 for event in roomInfo.timeline.events:
                     if event.sender != self.mxid:
-                        # if isinstance(event, nio.MegolmEvent):
-                        # event is encrypted
-                        # event = await self.connection.decrypt_event(event)
```

Comment:
Is this done for us elsewhere?

Reply:
The sync function decrypts the events

Reply:
In which case we should delete this block.